operator/Dockerfile

```diff
@@ -46,25 +47,17 @@ COPY operator/build/status.sh status.sh
 RUN git init && git add .
 
 # Build the operator binary.
-RUN GOOS=$(go env GOOS) scripts/go-build.sh operator
+RUN GOOS=$(go env GOOS) CGO_ENABLED=0 scripts/go-build.sh operator
 # Copy the operator binary to a location from where it will be taken into the final image.
 RUN cp -a bin/$(go env GOOS)/operator stackrox-operator
 
-# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
-FROM registry.access.redhat.com/ubi8-minimal:8.5
+FROM scratch
 
 ARG roxpath
 
 ARG ROX_IMAGE_FLAVOR
 ENV ROX_IMAGE_FLAVOR=${ROX_IMAGE_FLAVOR}
 
-RUN microdnf update && microdnf clean all && rm -rf /var/cache/yum/*
-
 COPY --from=builder ${roxpath}/stackrox-operator /usr/local/bin/
 
-# The following are numeric uid and gid of `nobody` user in UBI.
-# We can't use symbolic names because otherwise k8s will fail to start the pod with an error like this:
-# Error: container has runAsNonRoot and image has non-numeric user (nobody), cannot verify user is non-root (pod: "stackrox-operator-controller-manager-75bc744454-bkbjr_stackrox-operator-system(49874aae-2695-4d3a-afd3-8723914d2af5)", container: manager)
-USER 65534:65534
```

Comment:
@msugakov Why do we need that? I can add `/etc/passwd` to the image but I'm not sure if that's necessary

Reply:
You don't need `/etc/passwd`. When leveraging numeric uid:gid, the user doesn't need to exist in `/etc/passwd`.
If `USER` is not specified, the container will run as root. Maybe we override this in Kubernetes manifests, but still it is good to have run-as-user defined in the Dockerfile.